src/deadline/blender_submitter/addons/deadline_cloud_blender_submitter/open_deadline_cloud_dialog.py

```diff
@@ -9,11 +9,11 @@
 from deadline.client.job_bundle._yaml import deadline_yaml_dump
 from deadline.client.job_bundle.submission import AssetReferences
 from deadline.client.ui.dialogs.submit_job_to_deadline_dialog import SubmitJobToDeadlineDialog
-from deadline_cloud_blender_submitter import blender_utils as bu
-from deadline_cloud_blender_submitter import sanity_checks as sc
-from deadline_cloud_blender_submitter import scene_settings_widget as ssw
-from deadline_cloud_blender_submitter import template_filling as tf
-from deadline_cloud_blender_submitter._version import version_tuple as adaptor_version_tuple
+from . import blender_utils as bu
+from . import sanity_checks as sc
+from . import scene_settings_widget as ssw
+from . import template_filling as tf
+from ._version import version_tuple as adaptor_version_tuple
```

Comment:
Do you know why the imports needed to be reconfigured?

Reply:
The underlying reason, no. What I encountered was that the `deadline_cloud_blender_submitter` module couldn't be discovered by unit tests, before this change none of the tested modules imported from it. I can get you the error message in a second if you'd like

Reply:
```
    from deadline_cloud_blender_submitter import template_filling as tf
E   ModuleNotFoundError: No module named 'deadline_cloud_blender_submitter'
```
is from the test logs.